src/charactersheet/services/dm/weather.js

```diff
@@ -0,0 +1,73 @@
+const MUNDANE_WEATHER = [
+    'Light Wind',
+    'Fog',
+    'Hail',
+    'Heavy Snow',
+    'Rain',
+    'Sleet',
+    'Snow',
+    'Cold',
+    'Extreme Cold',
+    'Extreme Heat',
+    'Severe Cold',
+    'Severe Heat',
+    'Very Hot',
+    'Hurricane',
+    'Light Wind',
+    'Moderate Wind',
+    'Severe Wind',
+    'Strong Wind',
+    'Tornado',
+    'Windstorm',
+    'Blizzard',
+    'Duststorm',
+    'Greater Duststorm',
+    'Hurricane',
+    'Severe Thunderstorm',
+    'Snowstorm',
+    'Thunderstorm',
+];
+
+const ARCANE_WEATHER = [
```

Comment:
If this is the un-used functionality, my 2 cents is that it's not included in this PR as to not create tech debt/un-used code. Alternatively, an issue could be created that says to implement arcane weather.
I'm not demanding this change, just an idea.
Thoughts?

Reply:
I'm +0. I can remove it easy.